virt/virt-2-5-release-notes.adoc

```diff
@@ -53,6 +53,9 @@ The SVVP Certification applies to:
 //CNV-7295 - Download virtctl binary from the CLI Tools page
 * You can now download the `virtctl` client from the xref:../virt/install/virt-installing-virtctl.adoc#virt-installing-virtctl-client-web_virt-installing-virtctl[*Command Line Tools* page in the web console].
 
+//CNV-5783 - VM migration (RHV) - SRIOV support
+* You can now import a virtual machine with a Single Root Input/Output Virtualization (SR-IOV) network interface from Red Hat Virtualization.
```

Comment:
Looking around elsewhere in the repo, it looks like we're mostly spelling it "Single Root I/O Virtualization".

Reply:
Fixed.